src/Templates/Microsoft.NetCore.VB.ProjectTemplates/ClassLibrary.vstemplate

```diff
@@ -2,7 +2,7 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
     <Name Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="9"/>
-    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="6"/>
+    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="10"/>
```

Comment:
After digging into this.  The current implementation is correct.  The resource strings we want are only contained in the C# project system package `{860A27C0-B665-47F3-BC12-637E16A1050A}`.  These strings are not duplicated across the F# and VB project system packages